server/backend/database/mongo/client.go

```diff
@@ -90,6 +90,14 @@ func (c *Client) Close() error {
 	return nil
 }
 
+// DropDatabase drop mongodb table (only use in test cleanup)
+func (c *Client) DropDatabase(databaseName string) error {
+	if err := c.client.Database(databaseName).Drop(context.Background()); err != nil {
+		return fmt.Errorf("drop database(%s): %w", databaseName, err)
+	}
+	return nil
+}
+
```

Comment:
this code is only used in testcase for cleanup test database, such as `test-yorkie-meta-1678319278`